demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -346,6 +348,29 @@ def check_script_id(self, script_id_used_in_task, id_set_scripts):
         """
         return any([script_id_used_in_task in id_set_dict for id_set_dict in id_set_scripts])
 
+    def check_integration_command(self, integration_id_used_in_task, id_set_integrations,
+                                  empty_integration_id=True):
```

Comment:
maybe change empty_integration_id to command_without_brand? or something like that?

Reply:
You are right, it is better